src/main/java/mate/academy/spring/service/dto/mapping/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.service.dto.mapping.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    public OrderResponseDto toDto(Order order) {
+        return new OrderResponseDto(
```

Comment:
It seems to me that it is better to use setters ... in large constructors it is very easy to get confused and set the wrong value